src/static/js/modules/jquery/cf_formValidator.js

```diff
@@ -24,37 +24,59 @@ var _validator = {
     onSuccess: function() {}
   },
 
+  _skipType: function( elem, type ) {
+    var typeAttr = elem.attr( 'type' );
+    var skips = {
+      required:   !elem.prop( 'required' ),
+      email:      typeAttr !== 'email',
+      radiogroup: typeAttr !== 'radio',
+      checkgroup: typeAttr !== 'checkbox'
+    };
+    return skips[type];
+  },
+
   // Validate an individual input for each of the set types
-  // @param   {object} $input The jQuery object of the input
-  // @param   {string} value  The evaluated value of the input
-  // @returns {object}        The status of each of the tested types
-  _validateInput: function( $input, value ) {
+  // @param   {object} elem The jQuery object of the input
+  // @returns {object}      The status of each of the tested types
```

Comment:
As we chatted about, looks like jsdocs and eslint needs these comments as multi-line not single-line comment blocks.